### ISSUE TITLE
Accessibility: Axe issues triggered by <label> role in Va-Checkbox component

### ISSUE THREAD

**Issue by harshil1793** [department-of-veterans-affairs/vets-design-system-documentation#3515]
# Bug Report

- [ ] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened
Axe accessibility checks are flagging issues related to the role assigned to a <label> tag within the Va-Checkbox component. These issues might indicate potential accessibility violations.

https://github.com/department-of-veterans-affairs/component-library/blob/81958d75023c8d4fc43491355bdb0245c02d334e/packages/web-components/src/components/va-checkbox/va-checkbox.tsx#L239

Slack support request: https://dsva.slack.com/archives/CBU0KDSB1/p1731019140980939

## What I expected to happen
The Va-Checkbox component should pass all Axe accessibility checks without errors, ensuring proper roles and attributes are assigned to the <label> tag for compliance with accessibility standards.

## Reproducing

- Formation version:
- Device: (e.g. iPhone 8, Macbook)
- Browser: (e.g. Firefox, IE 11)

Steps to reproduce:

1. 
2.
3.
4.

## Urgency

How urgent is this request? Please select the appropriate option below and/or provide details

- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [ ] This bug is not blocking any work
- [ ] Other

**Comment by Mottie** [comment]
This appears to have been fixed in https://github.com/department-of-veterans-affairs/component-library/pull/1442

**Comment by amyleadem** [comment]
@caw310 
Confirming that the `va-checkbox` label no longer receives `"role="checkbox"`. This should be safe to close.
